pom.xml

```diff
@@ -29,12 +29,12 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.24</version>
+    <version>4.12</version>
     <relativePath />
   </parent>
 
   <artifactId>ssh-agent</artifactId>
-  <version>1.21-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

Comment:
Did you use `mvn incrementals:incrementalify`? You forgot to commit the `.mvn/` directory if so.

Reply:
Still missing `.mvn/extensions.xml`.

Reply:
Feel free to add that one file instead of commenting 2y old already merged PR ;-)
At the same time, might be just as well good to consider adding this file globally to all repositories under /jenkinsci org - I don't think it would harm them even if they're not using incrementals at the moment?

Reply:
> Feel free to add that one file

Already done in #85.

> consider adding this file globally to all repositories

Could, but probably more confusing than helpful, since `mvn incrementals:incrementalify` expects to add it.